core/src/lib.rs

```diff
@@ -48,6 +48,7 @@ impl Default for Options {
 }
 
 /// JSTime Instance.
+#[allow(clippy::upper_case_acronyms)]
```

Comment:
@steveklabnik do you have a suggested practice for ignoring clippy rules that are only on nightly?

Reply:
I don't know, I haven't ever done that, personally.

Reply:
figured it out! Needed to go a level deeper.

```
#[allow(clippy::unknown_clippy_lints)]
#[allow(clippy::upper_case_acronyms)]
```